extension/memorylimiterextension/factory.go

```diff
@@ -0,0 +1,35 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package memorylimiterextension // import "go.opentelemetry.io/collector/extension/memorylimiterextension"
+
```

Comment:
Please add this pragma to have a few things auto generated
```
//go:generate mdatagen metadata.yaml
```

See https://github.com/open-telemetry/opentelemetry-collector/blob/main/extension/ballastextension/factory.go#L4C1-L4C1 for the reference

Reply:
done